azavea/raster-foundry
# Show scenes already added to project

## Overview

This PR adds some visual cues to indicate that a scene has already been added to the current project within the editor.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

<img width="378" alt="screen shot 2017-04-20 at 5 59 05 pm" src="https://cloud.githubusercontent.com/assets/2442245/25254383/4ea3762c-25f3-11e7-8555-b27b3f74bca8.png">

## Notes

It feels like there is some ugliness (the `isInProject` parameter in particular) caused by the scene item component being used in two very different contexts.

## Testing Instructions

 * Open a project in the editor
 * Add some scenes and notice that they become grayed out and the option to select them goes away
 * Experiment with deleting and adding scenes to ensure the list state is correct

Closes #1476
